### ISSUE TITLE
Component doesn't work in the Authoring Tool

### ISSUE THREAD

**Issue by simondate** [NayanKhedkar/adapt-sentenceOrdering#1]
The component can be uploaded fine but when you try to create an instance of it this error appears `buildSchema - Cannot read property 'inputType' of undefined`

**Comment by andrelung** [comment]
In properties.schema you may change line 216 from "position" to "items". This fixed it for me
![screen](https://user-images.githubusercontent.com/4161059/36856314-00c5fc88-1d76-11e8-84f0-3d3ade8c611e.jpg)

**Comment by SteveX** [comment]
Using Authoring 0.5.0, Changing "position" to "items" fixes something in that I can create and display a component, but get a javascript error when submitting an answer.
TypeError: undefined is not an object (evaluating 'feedback.title') - getFeedbackTitle --- questionModel.js

**Comment by barryjenkin** [comment]
Was anyone able to get this working? In AT 0.5.0 I was able to install and create items (response options) in the component having made the above change to properties.schema but am getting incorrect on every attempt. In absence of feedback fields (correct, partly correct, etc), I've been using in conjunction with assessment / assessmentResults instead. In SCORM every combination of responses is marked incorrect despite setting an order. Does it need updating for questionModel.js?

**Comment by SteveX** [comment]
This version is working for me in 0.5.0 and 0.6.0, but I haven't tried anything with scorm.
https://github.com/BristolUniDigitalEducation/adapt-sentenceOrdering

**Comment by barryjenkin** [comment]
Ah thanks! Is yours giving you feedback? I tested on a new course and it seems to be struggling with 'position' variable on submit. Were you using with the latest assessment / assessmentResults?

![screen shot 2018-11-30 at 10 13 12 am](https://user-images.githubusercontent.com/24560844/49258848-049f9980-f48b-11e8-926e-7f81867af0b1.png)

**Comment by SteveX** [comment]
Yes, feedback works fine for us. I'm not using assessment at all, I just made it work with what we needed.
